policy/release_manager.md

```diff
@@ -0,0 +1,54 @@
+# Release manager role
+
+## Appointment
+
+A release manager (RM) is appointed for a specific upcoming major or minor release (i.e., x.y.z -> x.y+1.0 or x.y.z -> x+1.0.0).
+
+A RM may and should step down at any time, when they cannot fulfill the associated obligations.
+In this case, a new RM is appointed.
+
+The appointment of an RM ends with the declaration of the respective release's end-of-life by the RM.
+
+## Responsibilities
+
+Immediately after the previous release was made, the RM takes over decision making regarding merges into the integration branch for all developments targeting the respective release.
+Such merges can be made as soon as, but no earlier than, the RM indicates approval.
+
+It is the RM's responsibility to act on any merge request in a timely manner.
+
+An RM is not expected to be an expert in all techniques, features, and parts of the code base.
+Consequently, an RM should seek feedback prior to approving merge requests whenever necessary.
+
+The RM declares the time of a feature freeze for the upcoming release.
+
+The RM decides on the length of the consolidation period following a feature freeze, and decides which kinds of changes are still acceptable.
+
+The RM decides on the number and timing of potential release candidates, and on the timing of the final release.
+
+After the initial release, the RM takes over the decision making for the respective maintenance branch in the same fashion as for the prior development towards the initial release.
+The RM decides on the nature of changes qualified for maintenance releases, and their timing.
+
+
+## Assistant release manager role
+
+In addition to an RM there is an assistant RM to prevent disruption in cases where the RM temporarily cannot act on their duties.
```

Comment:
who is assigning/deciding on being an assistant RM?  is it like a president+prime to come into position at the same time?

Should it just be "next RM" to serve as "assistant RM" for current cycle, thus removing this ambiguity and possibly gaining some needed training?  if so

```suggestion
RM appointed for the next cycle serves as an assistant RM to prevent disruption in cases where the RM temporarily cannot act on their duties.
```